### ISSUE TITLE
logger_entity table error on update Data too long for column 'name'.

### ISSUE THREAD

**Issue by artemdotsenko** [ymcatwincities/openy#833]
During update from platform version 8.1.4 to 8.1.5 faced with next error:
```
String data, right truncated: 1406 Data too long for column 'name' at row 1: INSERT INTO {logger_entity} (type, uuid, langcode, user_id, name, status,
created, changed, data) VALUES (:db_insert_placeholder_0, :db_insert_placeholder_1, :db_insert_placeholder_2,
:db_insert_placeholder_3, :db_insert_placeholder_4, :db_insert_placeholder_5, :db_insert_placeholder_6,
:db_insert_placeholder_7, :db_insert_placeholder_8); Array
(
    [:db_insert_placeholder_0] => openy_config_upgrade_logs
    [:db_insert_placeholder_1] => 74bf8e08-e43a-48fb-831e-eab04818b624
    [:db_insert_placeholder_2] => en
    [:db_insert_placeholder_3] => 0
    [:db_insert_placeholder_4] => core.entity_view_display.media.image.node_program_subcategory_teaser
    [:db_insert_placeholder_5] => 1
    [:db_insert_placeholder_6] => 1509011868
    [:db_insert_placeholder_7] => 1509011868
    [:db_insert_placeholder_8] =>
a:1:{i:0;s:68:"core.entity_view_display.media.image.node_program_subcategory_teaser";}
)
 in Drupal\Core\Entity\Sql\SqlContentEntityStorage->doSaveFieldItems() (line 850 of
/var/www/docroot/core/lib/Drupal/Core/Entity/Sql/SqlContentEntityStorage.php).
Failed to save logger entity.
```

Looks like length in 50 characters is not enough:
![logged_entity](https://user-images.githubusercontent.com/4688363/32047915-819b9526-ba15-11e7-8aab-7731f19c9c42.png)

**Comment by artemdotsenko** [comment]
There was open PR to fix this error https://github.com/ymcatwincities/openy/pull/578

**Comment by alexschedrov** [comment]
Related to #838.

**Comment by alexschedrov** [comment]
Fixed in #838.
